Encryption.md

```diff
@@ -133,48 +159,144 @@ Algorithms section. The length of the encryption buffer (a 4-byte little endian)
 written to the output stream, followed by the buffer itself.
 
 The column pages (data and dictionary) are encrypted after compression. For each page, 
-the encryption buffer is comprised of an IV, ciphertext and (in case of AES_GCM_V1) of a 
+the encryption buffer is comprised of an IV (nonce), ciphertext and (in case of AES_GCM_V1) of a 
 tag, as described in the Algorithms section. Only the buffer is written to the output 
-stream - not need to write the length of the buffer, since the length (size of the page after
+stream - no need to write the length of the buffer, since the length (page size after
 compression and encryption) is kept in the page headers.
 
```

Comment:
PageHeaders (and Pages in case of GCM cipher) are encrypted with an AAD concatenated from column offset and user-supplied AAD (if present). I'll add description.